crates/pixi_manifest/Cargo.toml

```diff
@@ -44,7 +45,7 @@ rattler_solve = { workspace = true, optional = true }
 
 # TODO: Remove these dependencies
 console = { workspace = true }
-miette = { workspace = true }
+miette = { workspace = true, features = ["fancy-no-backtrace"] }
```

Comment:
Is there a result you could share where this would change the result?

Reply:
it is used in tests to disable colors